packages/flutter_tools/templates/plugin/README.md.tmpl

```diff
@@ -4,7 +4,11 @@
 
 ## Getting Started
 
-For help getting started with Flutter, view our online
-[documentation](https://flutter.io/).
-
-For help on editing plugin code, view the [documentation](https://flutter.io/developing-packages/#edit-plugin-package).
+This project is a starting point for a
+[Flutter plug-in](https://flutter.io/developing-packages/),
```

Comment:
I suggest we s/[Flutter plug-in]/[plugin package]/ to be consistent with the the package type phrasing here:
https://flutter.io/docs/development/packages-and-plugins/developing-packages#package-types

Reply:
Do we want to link deeply to https://flutter.io/docs/development/packages-and-plugins/developing-packages#developing-plugin-packages ?

Reply:
Thanks! Updated name. 

FWIW, I decided against a deep link because it seemed less resilient to authors unwittingly changing the doc. I link to the redirect also for that reason, knowing that this URL should be stable even if we restructure the IA.